backend/contracts/door.sol

```diff
@@ -44,25 +44,24 @@ contract doorManager is Ownable {
     }
 }
 
-//pragma solidity >=0.4.22 <0.6.0;
-
 contract Door is Ownable, Initializable {
 
     bool started;
     bool ended;
 
-    enum TicketValues { None, RSVPD, Attended }
+    enum AttendanceTypes { NONE, REGISTERED, ATTENDED }
     uint ticketPrice;
-    mapping(address => TicketValues) public ticketHolders;
+    mapping(address => AttendanceTypes) public tickets;
+    address[] public registerees;
+    address payable[] public attendees;
 
 
     modifier hasNoTicket() {
-        require(ticketHolders[msg.sender] == TicketValues.None, 'you have no ticket');
+        require(tickets[msg.sender] == AttendanceTypes.NONE, 'you have no ticket');
         _;
     }
 
     function initialize(uint256 _price) public initializer payable  {
-        //owner = msg.sender;
          ticketPrice = _price;
     }
 
```

Comment:
normally use require()
documentation: "The assert function should only be used to test for internal errors, and to check invariants. "